pkg/booleanpolicy/evaluator/evaluator_test.go

```diff
@@ -1344,7 +1344,7 @@ func TestDifferentBaseTypes(t *testing.T) {
 				FieldQueries: []*query.FieldQuery{
 					// This test will fail in three years, but if this test still matters then,
 					// we have other problems...
-					{Field: "BaseTS", Values: []string{">1000d"}},
+					{Field: "BaseTS", Values: []string{">2000d"}},
```

Comment:
Please change this as well as the comment to 1000y instead (or if that does not fit, 100y).

Reply:
I oversaw this one before merging, created a follow-up for this: #4255.
Sorry!